src/lib/integration_test_cloud_engine/kubernetes_network.ml

```diff
@@ -78,6 +78,19 @@ module Node = struct
       @ [ "exec"; "-c"; container_id; "-i"; pod_id; "--" ]
       @ cmd )
 
+  let run_in_pod ?(exit_code = 10) ?override_with_pod_id ~cmd t =
```

Comment:
what is the difference between this function and `run_in_container`?  from what I can tell, `run_in_container` uses `kubectl exec -c <container_id> -i <pod_id>`, while this new `run_in_pod` function simply omits the `-c` flag.  doesn't that just mean it gets run in the default container?  I think generally it is good to be explicit as to which container you're running things in

Reply:
Apologies this commit was some leftover and was reverted